### PR TITLE
[#m8xpmt] Create Navbar

### PR DESCRIPTION
**Pull request by allispeck**
most styles applied, mobile responsive.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="stylesheet" href="/"
```
Comment (BrayonCrayon) [inline public/index.html]: Could we remove this tag?

**Review by BrayonCrayon — CHANGES_REQUESTED** [review]

**Comment by BrayonCrayon** [comment]
Good work!

**Comment by allispeck** [comment]
removed it!

File: src/layout/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import {NavLink} from "react-router-dom";
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome'
+import {faBars, faDog} from "@fortawesome/free-solid-svg-icons";
+import {MOBILE_SCREEN_SIZE} from "../Constants";
+
+
+const Navbar = () => {
+    const [show, setShow] = React.useState(false);
+
+    let w = document.documentElement.clientWidth || window.innerWidth;
```
Comment (BrayonCrayon) [inline src/layout/Navbar.js]: @allispeck, can we use Tailwindcss's css breakpoints instead? Might run into some issues later on that could impact how you would construct your components :)
https://tailwindcss.com/docs/responsive-design

Comment (BrayonCrayon) [reply src/layout/Navbar.js]: Awesome!

**Review by BrayonCrayon — APPROVED** [review]

**Pull request merged**